.gitignore

```diff
@@ -1,3 +1,4 @@
 /vendor
 /composer.lock
 /phpunit.xml
+.idea
```

Comment:
You can add this to your global .gitignore.

This repository will not contain the list of every IDE's tmp folders.